server/service/src/sync/translations/name.rs

```diff
@@ -122,13 +132,21 @@ pub struct LegacyNameRow {
 
     #[serde(rename = "isDeceased")]
     pub is_deceased: bool,
-    // TODO not in mSupply:
-    //pub om_created_datetime: Option<NaiveDateTime>,
-    //pub om_gender: Option<Gender>,
+    // #[serde(rename = "om_created_datetime")]
```

Comment:
👀  for #1220, can ignore here

Reply:
I worked on in:

1220-name-integration-2

but couldn't get the integration test to work, because I never received a patient...

Reply:
Oh should I push my changes for the pull in there? >.> I have a working branch

Reply:
Could work my changes into your branch? think having two branches only leads to merge conflicts in this case... I branched off from `develop` though but maybe cherry picking my commits without conflicts? otherwise you could just do it manually...